### PR TITLE
[Merged by Bors] - vestingen: make it easier to use and support bulk execution with config

### PR DESCRIPTION
**Pull request by dshulyak**
examples:

./vestingmulti/ -c vestingmulti/example.csv -out=go
./vestingmulti/ -c vestingmulti/example.json 
./vestingmulti/ -c vestingmulti/example.json -out=go

csv decodes columns in this order:
1. total amount
2. required signatures
3. total keys
4. next as many columns as specified in prev column (one key - one column)

output is either in json or as a golang map, so that it can be copied as is to config 

```json
{"address":"sm1qqqqqqr3cj4agq0g6evj5cxma5r7arlmqzjvr4cqv9h6l","balance":10000000000}
{"address":"sm1qqqqqqzev03u56u7u4228d23d7p43ujluzp6yxs6uc0ej","balance":100000000000}
```

```go
func MainnetAccounts() map[string]uint64 {
    "sm1qqqqqqr3cj4agq0g6evj5cxma5r7arlmqzjvr4cqv9h6l": 10000000000,
    "sm1qqqqqqzev03u56u7u4228d23d7p43ujluzp6yxs6uc0ej": 100000000000,
}
```

### CODE REVIEW

File: genvm/cmd/vesting/README.md [hunk 1]
```diff
@@ -15,32 +16,65 @@ This tool generates an address for vesting and vault accounts, based on the set
...
+./smcli wallet create
+```
+
+
+3. Extract public keys from wallet for each one.
+
+```
+./smcli wallet read /home/dd/.spacemesh/wallet_2023-07-12T06-14-00.086Z.json -f
+...
+./smcli wallet read /home/dd/.spacemesh/wallet_2023-07-12T06-43-19.316Z.json -f
```
Comment (countvonzero) [inline genvm/cmd/vesting/README.md:35]: nice

File: genvm/cmd/vestingmulti/example.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "keys": [
+        "33f8d3b3a9fd558ccef98e9693b6b264e5a593ea096c8984012e94c6e7f72692",
+        "93f801839a7e709f4c270370665ee71b7d86b5f65cb816559503b311d3213962"
+    ], 
+    "required": 1, 
+    "total": 100000e9
+}
```
Comment (lrettig) [inline genvm/cmd/vestingmulti/example.json:8]:
```suggestion
},
```

**Review by countvonzero — APPROVED** [review]

**Review by lrettig — DISMISSED** [review]

File: genvm/cmd/vestingmulti/example.json (hunk 2 above)
Comment (dshulyak) [reply genvm/cmd/vestingmulti/example.json:8]: it is not an array, decoder goes through them as separate json objects.
it works as intended, you can check it

**Comment by dshulyak** [comment]
i will merge just in case if it will be useful for cross checking. we can drop it a later date if it will seem confusing

**Comment by dshulyak** [comment]
bors merge

**Pull request closed without merging**